lib/webapp.js

```diff
@@ -412,7 +412,9 @@ const staticHandlers = {
               loader.once('refreshFinish', (event) => {
                 respondToGetPlugins(req, res, webapp, type, webapp.plugins);
               });
-              loader.installPlugins(defs);
+              loader.installPlugins(defs).then( () => {
```

Comment:
Catch an error here.

Reply:
The error for bad checks in installPlugins get caught down stream. Success of installPlugins itself doesn't have a return condition, I just made it this way to make it async